packages/gateway/helpers.ts

```
@@ -0,0 +1,22 @@
+import { AUTH_ROLE, AUTH_USER_ID, AUTH_USERNAME } from '../server/src/shared/constants';
```

ESLint

[error] Missing file extension for "../server/src/shared/constants"

(import/extensions)